net/autossh/files/autossh.init

```diff
@@ -14,20 +15,15 @@ start_instance() {
 
 	[ "$enabled" = 1 ] || exit 0
 
-	export AUTOSSH_GATETIME="${gatetime:-30}"
-	export AUTOSSH_POLL="${poll:-600}"
-	service_start /usr/sbin/autossh -M ${monitorport:-20000} -f ${ssh}
+	procd_open_instance
+	procd_set_param command /usr/sbin/autossh -M ${monitorport:-20000} -f ${ssh}
```

Comment:
No idea about this.

-f means fork I think. But procd demands that services run in the foreground.

Reply:
You are right, I remove that. The process was hanging and stop/restart wasn't working correctly.